app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -130,11 +130,32 @@ def queue_signal(*args, deliver_on: nil)
     )
   end
 
-  def playbook
-    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook.find(options[:playbook_id])
+  def temp_configuration_script
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript.new(
```

Comment:
It feels like this should be cached or something, otherwise you'll get a new `ConfigurationScript` on each call.

Reply:
It is used only twice, and each time in a different instance because of they are off from the queue. There is no need to cache.